src/h5web/vis-packs/core/line/ErrorBars.tsx

```diff
@@ -13,24 +13,25 @@ interface Props {
 function ErrorBars(props: Props) {
   const { barsSegments, capsPoints, color, visible } = props;
 
-  const barsGeometry = useUpdate<BufferGeometry>(
-    (geometry: BufferGeometry) => geometry.setFromPoints(barsSegments),
-    [barsSegments]
-  );
-  const capsGeometry = useUpdate<BufferGeometry>(
-    (geometry) => geometry.setFromPoints(capsPoints),
-    [capsPoints]
-  );
+  const barsGeometry = useRef<BufferGeometry>(null);
+  useLayoutEffect(() => {
+    barsGeometry.current?.setFromPoints(barsSegments);
+  }, [barsGeometry, barsSegments]);
+
+  const capsGeometry = useRef<BufferGeometry>(null);
+  useLayoutEffect(() => {
+    capsGeometry.current?.setFromPoints(capsPoints);
+  }, [capsGeometry, capsPoints]);
```

Comment:
`useUpdate` was removed. This is the recommended alternative. As mentioned in the PR description, I think there may be a way to refactor this.